randovania/games/fusion/logic_database/Sector 4 AQA.txt

```diff
@@ -944,6 +960,8 @@ Extra - room_id: [23]
           Any of the following:
               Can Use Springball
               Gravity Suit and Can Use Bombs
+              # Mid-Air Morph: https://youtu.be/zyazrYLRuSk
+              Mid-Air Morph (Intermediate)
```

Comment:
does the slowness of the water make this more lenient? since usually this height would be advanced

Reply:
it was pretty free yes, not beginner but definitely easier in water